api/Document.json

```
@@ -3899,10 +3899,10 @@
             "description": "<code>copy</code> command",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": "43"
```

I was sort of surprised that this wasn't supported earlier, so I read [the Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=437908) and it appears to have shipped in [version 42](https://storage.googleapis.com/chromium-find-releases-static/07d.html#07d72f48a5603798519b459eed83eb4a68b10069), not 43. This also lines up better with the Opera numbers.

```suggestion
                "version_added": "42"
```